forage-android/src/main/java/com/joinforage/forage/android/ui/ForagePANEditText.kt

```diff
@@ -55,17 +56,54 @@ class ForagePANEditText @JvmOverloads constructor(
         context.obtainStyledAttributes(attrs, R.styleable.ForagePANEditText, defStyleAttr, 0)
             .apply {
                 try {
+                    val focusedBoxStrokeColor = getColor(
+                        R.styleable.ForagePANEditText_android_strokeColor,
+                        getThemeAccentColor(context)
+                    )
+
+                    val focusedBoxStrokeWidth = getDimension(
+                        R.styleable.ForagePANEditText_boxStrokeWidthFocused,
+                        3f
+                    )
+
+                    val strokeWidth = getFloat(
+                        R.styleable.ForagePANEditText_android_strokeWidth,
+                        3f
+                    )
+
                     val textInputLayoutStyleAttribute =
                         getResourceId(R.styleable.ForagePANEditText_textInputLayoutStyle, 0)
                     val textColor =
                         getColor(R.styleable.ForagePANEditText_android_textColor, Color.BLACK)
 
                     val textSize = getDimension(R.styleable.ForagePANEditText_android_textSize, -1f)
 
+                    val defRadius = resources.getDimension(R.dimen.default_horizontal_field)
+                    val focusedBoxCornerRadius =
+                        getDimension(R.styleable.ForagePANEditText_cornerRadius, defRadius)
+
+                    val boxCornerRadiusTopStart = getBoxCornerRadiusTopStart(focusedBoxCornerRadius)
+                    val boxCornerRadiusTopEnd = getBoxCornerRadiusTopEnd(focusedBoxCornerRadius)
+                    val boxCornerRadiusBottomStart = getBoxCornerRadiusBottomStart(focusedBoxCornerRadius)
+                    val boxCornerRadiusBottomEnd = getBoxCornerRadiusBottomEnd(focusedBoxCornerRadius)
+
                     textInputLayout =
                         TextInputLayout(context, null, textInputLayoutStyleAttribute).apply {
                             layoutParams =
                                 LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.WRAP_CONTENT)
+
+                            boxStrokeColor = focusedBoxStrokeColor
+                            boxStrokeWidthFocused = focusedBoxStrokeWidth.toInt()
```

Comment:
I'm not very familiar with a lot of Android styling. How come we have a `focusedBoxStrokeWidth` and `strokeWidth` but we have only `focusedBoxStrokeColor` and no `strokeColor`? Asking so I can learn!

Reply:
Good question! I think(based on my research) that Android TextInputLayout does not provide a direct method to set the stroke color programmatically. The stroke color inside the TextInputLayout is typically controlled through XML styles. 
So, stroke width and color, and unfocused stroke width can be set programmatically. Unfocused stroke color can be changed through `styles.xml`. 
Also, I added one-line comments above those setters for better understanding what we have and what we can change through ForagePANEditText.

Reply:
really solid explanation - appreciate it a lot!